docs/source/models/gbd2017/causes/ischemic_heart_disease/ischemic_heart_disease.rst

```diff
@@ -0,0 +1,68 @@
+.. _2017_cause_ischemic_heart_disease:
+
+===================
+Ischemic Heart Disease
+===================
```

Comment:
The heading underline must be at least as long as the title for valid `.rst`. It should look like this:

```
======================
Ischemic Heart Disease
======================
```

Reply:
I made this update. Thanks!